fix #1275 (#1308)
V4Fire/Client: CHANGELOG.md

_Note: Gaps between patch versions are faulty, broken or test releases._

## v4.0.0-beta.?? (2024-06-??)

#### :bug: Bug Fix

> * Fix hookChange event processing for activate/deactivate component `core/component/init`

Reviewer: Надо сделать более понятное описание

_Note: Gaps between patch versions are faulty, broken or test releases._

## v4.0.0-beta.?? (2024-06-??)

#### :bug: Bug Fix

> * Fixed a typo in the hookChange event name for processing activate/deactivate component `core/component/init`

Reviewer 2: ```suggestion
* Fixed a typo in the hookChange event name for processing activate/deactivate in component `core/component/init`
```

> * Fix component deactivation order, children must be deactivated first `core/component/init`

Reviewer 2: ```suggestion
* Fixed the component deactivation order: children must be deactivated first `core/component/init`
```

> * Fix hookChange event should not bubble `bDynamicPage`

Reviewer 2: ```suggestion
* Fixed an issue where the hookChange event should not bubble in `bDynamicPage`
```
